lib/dom.js

```diff
@@ -1114,6 +1115,16 @@ declare class HTMLLabelElement extends HTMLElement {
     control: HTMLElement | null;
 }
 
+declare class HTMLLinkElement extends HTMLElement {
+    crossorigin: 'anonymous' | 'use-credentials';
```

Comment:
i think this can also be `void`. I ran `document.querySelector("link").crossorigin` on flowtype.org and got undefined in Chrome.